2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
+the management for these workshops is currently being done by 1-2 persons and
+in very rough conditions (via files and text notes).  That's why Amy was
+created in December 2014: to help Software Carpentry admins schedule
+workshops, assign instructors to workshops, and even coordinate training for
+new instructors.
+
+The aim of this project is to continue enhancing Amy.  Many people
+[suggested great ideas](https://github.com/swcarpentry/amy/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement+-label%3Aessential) for Amy,
+some even [implemented them](https://github.com/swcarpentry/amy/pulls?q=is%3Apr+is%3Aopen+sort%3Acreated-asc), but not much was actually
+merged in -- and, as a result, in March
+[we can see a drop in new commits](https://github.com/swcarpentry/amy/graphs/contributors).
```

Comment:
Hooray for links backing up your claims :).  This is very nice, although personally I'd just drop in a UTF-8 em dash (—) ;).